docs/index.md

```diff
@@ -839,6 +839,7 @@ outcome | object | The outcome of the transaction (what effects it had).
 *outcome.* ledgerVersion | integer | The ledger version that the transaction was validated in.
 *outcome.* indexInLedger | integer | The ordering index of the transaction in the ledger.
 *outcome.* deliveredAmount | [amount](#amount) | *Optional* For payment transactions, it is impossible to reliably compute the actual delivered amount from the balanceChanges due to fixed precision. If the payment is not a partial payment and the transaction succeeded, the deliveredAmount should always be considered to be the amount specified in the transaction.
+*outcome.* deliveredAmount | string | *Optional* The delivered amount is reported as 'unavailable' for partial payment transactions before 2014
```

Comment:
It has to be optional because it can only be present for successful Payment transactions, part of the reason they created the "delivered_amount" decorator because "DeliveredAmount" was not always present, so it was potentially confusing to understand exactly how much was transferred....this way it would always be present when *some* amount is transferred, even if it can't explictly state what that amount is.